e2e/test/scenarios/embedding/embedding-full-app.cy.spec.js

```diff
@@ -315,6 +326,64 @@ describeEE("scenarios > embedding > full app", () => {
 
       cy.findByTestId("main-logo").should("be.visible");
     });
+
+    it("should have parameters header occupied the entire horizontal space when visiting a dashboard via navigation (metabase#30645)", () => {
+      const dashboardDetails = {
+        name: "interactive dashboard embedding",
+        parameters: [
+          {
+            id: "50c9eac6",
+            name: "ID",
+            slug: "id",
+            type: "id",
+          },
+        ],
+      };
+      cy.createDashboard(dashboardDetails).then(
+        ({ body: { id: dashboardId } }) => {
+          const card = createTextCard({
+            col: 0,
+            row: 0,
+            size_x: 6,
+            size_y: 20,
+            text: "I am a very long text card",
+          });
+          updateDashboardCards({ dashboard_id: dashboardId, cards: [card] });
+        },
+      );
+
+      visitFullAppEmbeddingUrl({ url: "/" });
+
+      cy.log("Navigate to a dashboard via in-app navigation");
+      navigationSidebar().findByText("Our analytics").click();
+      cy.findByRole("main").findByText(dashboardDetails.name).click();
+      closeNavigationSidebar();
+      navigationSidebar().findByText("Our analytics").should("not.be.visible");
```

Comment:
ensure that the sidebar is close, so we could assert that the dashboard parameter's `x` is `0` for simplicity.

Noted that the behavior is different on 0.46.1 where after we navigate to the dashboard the side nav is closed automatically, so we wouldn't need to close the sidebar when testing on 0.46.1.